tidb-cloud/serverless-tier-limitations.md

```diff
@@ -19,6 +19,12 @@ We are constantly filling in the feature gaps between Serverless Tier and Dedica
     - The compute resource is limited to 1 vCPU and 1 GiB RAM.
     - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free Serverless Tier might be changed.
 
+## SQL
+
+- Using [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) to improve the speed of index creation is not supported for Serverless Tier clusters.
```

Comment:
This is too details. do not need to show here.

Reply:
OK.  I'll remove this line for now.